libs/testing/src/integration.service.ts

```diff
@@ -58,5 +58,18 @@ export class IntegrationService {
     };
 
     await this.integrationRepository.create(pushFcmPayload);
+
+    const inAppPayload = {
```

Comment:
This was moved from here as it didn't make sense to me to have it isolated (and also thus we create the integration also for any environment we create during tests).
https://github.com/novuhq/novu/blob/b23c6c56c55ab9950495bde79caccecd75e46d4f/libs/testing/src/user.session.ts#L149